roles/docker/tasks/systemcontainer_crio.yml

```diff
@@ -1,18 +1,21 @@
 ---
 
 # TODO: Much of this file is shared with container engine tasks
+- set_fact:
```

Comment:
Why are we setting these facts like this?

Let's not use set_fact for this, this is what role defaults are for.